docs/WebsocketAPIV5.md

```diff
@@ -0,0 +1,235 @@
+## WebSocket protocol v5 details
+
+WebSocket Protocol v5 requires an Exasol version of at least 8.32.0. It follows the RFC 6455 document of the IETF.
+
+The Exasol connection server identifies the initial GET request by the client.
+This request contains information about the used protocol version.
+Depending on this information the matching login and protocol class is
+chosen.
+
+After the handshake the process is identical to a connection using the
+standard drivers like JDBC or ODBC: The connection server listens to
+incoming messages and forwards the requests to the database. 
+
+## Table of contents
+* [Changes](#changes)
+* [Command summary](#command-summary)
+* [Attributes: Session and database properties](#attributes-session-and-database-properties)
+* [Data Types: Type names and properties](#data-types-type-names-and-properties)
+* [Compression](#compression)
+* [Heartbeat/Feedback messages](#heartbeatfeedback-messages)
+* [Subconnections](#subconnections)
+
+## Changes
+
+| Date | Exasol Version | Change | Issue |
+| --- | --- | --- | --- |
+| 2024.09.05 | 8.32.0 | The latest update to Exasol enables the use of TIMESTAMP data types with various precisions. | |
```

Comment:
May I know, What does this date refers to?

Reply:
Date up publishing/release. They'll have to be adjusted.